task_1/plotting_utils/plot_distributions.py

```diff
@@ -0,0 +1,63 @@
+import os
+import glob
+import pandas as pd
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def plot_distributions(x1, x2, min_v=-300, max_v=800, balance=True, names=['test', 'train']):
+    """Plots the distribution between two variables of interest, x1 and x2.
+
+    :param x1: variable 1 (smallest number of samples - if balance = True)
+    :type x1: pandas.core.frame.DataFrame 
+
+    :param x2: variable 2  (lagestest number of samples - if balance = True)
+    :type x2: pandas.core.frame.DataFrame
+
+    :param names: list containing strings of dataset names (for plotting) in same order provided
+    :type names: list
+
+    :param balace: to balance the sets based on x1 dimension (dim 0), defaults to True
+    :type balace: bool, optional
+    """
+
+    if balance:
+        try:
+            # Check length of x1 <= x2
+            assert(x1.shape[0] <= x2.shape[0])
+
+            # Balance data
+            n_samples = np.shape(x1)[0]
+            x2 = x2.sample(n_samples)
+
+        except AssertionError as e:
```

Comment:
would be nicer if we don't impose this restriction of size for given sets, but just act accordingly to which is larger